Tools-Override/CloudTest.targets

```diff
@@ -428,10 +428,25 @@
       </ForUpload>
     </ItemGroup>
   </Target>
+  
+  <Target Name="CopyTestILCToRuntimeDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
```

Comment:
Why is this needed?  If it's a Helix run, we can just zip up the path without having to copy it recursively.  If it's a local run, you can run it from where ever it already is copied..